training_project/pages/login_page.py

```diff
@@ -0,0 +1,37 @@
+from selenium.webdriver.common.by import By
+
+from training_project.pages.base_page import BasePage
+
+
+class LoginPage(BasePage):
+    def __init__(self, browser):
+        super().__init__(browser)
+
+        self.username_name = (By.NAME, "username")
+        self.password_input = (By.NAME, "password")
+        self.login_button = (By.XPATH, '//*[@id="app"]/div[1]/div/div[1]/div/div[2]/div[2]/form/div[3]/button')
```

Comment:
It is bad idea to copy xpath from inspector. You depend on many interim divs. Try to write it manuall, you will be surprised how short it could be)
Same comment for other copy-pasted locators)